src/sharedServices/s3/readS3DataToString.ts

```diff
@@ -2,17 +2,11 @@ import { GetObjectCommand, S3Client } from '@aws-sdk/client-s3'
 import consumers from 'stream/consumers'
 import { Readable } from 'stream'
 import { getEnv } from '../../utils/helpers'
-import { logger } from '../logger'
 
 export const readS3DataToString = async (
   bucketName: string,
   fileKey: string
 ): Promise<string> => {
-  logger.info(`reading file from bucket`, {
```

Comment:
I think this is still a useful log to have, the key and bucket name would help spot if something was going wrong

Reply:
Not if it's only being used for one thing, which is to read the email recipient list